site/content/extend/plugins/overview/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Overview
+date: 2018-07-10T00:00:00-05:00
+subsection: Plugins (Beta)
+weight: 1
+---
+# Overview
+
+Plugins are defined by a manifest file and contain at least a server or web app component, or both. The [sample plugin](https://github.com/mattermost/mattermost-plugin-sample) is a starting point and illustrates the different components of a Mattermost plugin. A more detailed example is the [demo plugin](https://github.com/mattermost/mattermost-plugin-demo), which showcases many of the features of plugins.
+
+### Manifest
+The plugin manifest provides required metadata about the plugin, such as name and ID. It is defined in JSON or YAML. This is `plugin.json` in both the [sample](https://github.com/mattermost/mattermost-plugin-sample/blob/master/plugin.json) and [demo](https://github.com/mattermost/mattermost-plugin-demo/blob/master/plugin.json) plugins.
+
+See the [manifest reference](/extend/plugins/manifest-reference) for more information.
+
+### Server
+The server component of a plugin is written in Go and runs as a subprocess of the Mattermost server process. The Go code is centered around extending the [MattermostPlugin](https://godoc.org/github.com/mattermost/mattermost-server/plugin#MattermostPlugin) struct that contains an [API](/extend/plugins/server/reference/#API) and allows for the implementation of [Hook](/extend/plugins/server/reference/#Hooks) methods that enable the plugin to interact with the Mattermost server.
```

Comment:
grammar nit: `centre on` vs. `centre around`, but I suspect both of these would be unfamiliar to non-native English speakers. Perhaps just simply, `The Go code extends the MattermostPlugin struct that contains ...`

Reply:
Good point, will update